electron/src/plugins.ts

```diff
@@ -0,0 +1,18 @@
+import { Plugins } from './global';
+import { mergeWebPlugins, mergeWebPlugin, WebPlugin } from '../../core/src/web/index';
+
+export * from '../../core/src/web/browser';
```

Comment:
What are you trying to do here? Should we change something in Core so you don't have to do this?

Reply:
This is so if bundles it all together in an `electron-bridge.js` in the dist of `@capacitor/electron` this is then injected into the apps `index.html`. You can reference `@capacitor/core` in the app and it will all play nice together, see the test app I posted, and give building it a shot, Im quite horrible at explaining this.